currency/templates/currency/all_currencies.html

```diff
@@ -39,6 +41,14 @@ <h5>All currencies</h5>
                                 <td>{{currency.series}}</td>
                                 <td><img src="{{currency.thumbnail.url}}"></td>
                                 <td>{{currency.date_created}}</td>
+                                <td>
+                                    {% if user.is_authenticated %}
```

Comment:
How does the page look like when user is not logged in? Seems like this check should be higher up.